### PR TITLE
LL-7755 - ProgressLoader component improved and purple color fixed on dark theme

### PR DESCRIPTION
**Pull request by cgrellard-ledger**

### CODE REVIEW

File: packages/native/package.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "@ledgerhq/native-ui",
   "version": "0.6.6",
-  "private": true,
```
Comment (elbywan) [inline packages/native/package.json:4]: ⚠️ This line is there to prevent accidental publishing, could you revert the change?

Comment (elbywan) [reply packages/native/package.json:4]: (the publishing script takes care of formatting the package.json file: https://github.com/LedgerHQ/ui/blob/main/packages/native/scripts/prepareRelease.js#L25)

File: packages/native/src/components/Loader/index.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
-export { default as Loader } from "./ProgressLoader";
```
Comment (elbywan) [inline packages/native/src/components/Loader/index.ts:1]: this is a breaking change

File: packages/native/storybook/stories/Loader/ProgressLoader.stories.tsx [hunk 3]
```diff
@@ -1,16 +1,21 @@
 import React from "react";
 import { storiesOf } from "../storiesOf";
-import { number } from "@storybook/addon-knobs";
+import { number, color, boolean } from "@storybook/addon-knobs";
 import { action } from "@storybook/addon-actions";
-import Loader from "../../../src/components/Loader/ProgressLoader";
-import { Icons } from "../../../src/assets";
+import { Icons, ProgressLoader } from "src";
```
Comment (elbywan) [inline packages/native/storybook/stories/Loader/ProgressLoader.stories.tsx:5]: I think that a relative path is needed here otherwise storybook will fail to build.

**Review by LFBarreto — APPROVED** [review]
